src/code.l

```diff
@@ -2061,7 +2061,7 @@ RAWEND    ")"[^ \t\(\)\\]{0,16}\"
 					  endFontClass();
 					  BEGIN( Body );
   					}
-<Body,Bases>^[ \t]*"#"			{ 
+<*>^[ \t]*"#"			{ 
```

Comment:
Maybe `<Body,Bases>` is too limited, but `<*>` is too liberal. We don't want to handle preprocessor directives in comments or string literals for instance.

Reply:
I agree that `<*>` is a bit too liberal. It is hard to determine which states should be used, so I added just the state from this issue. Feel free to add other states.